src/Education/XamlGenerator/XamlOutputElementInfo.cs

```diff
@@ -20,40 +20,37 @@
 
 namespace SonarLint.VisualStudio.Education.XamlGenerator
 {
-    internal partial class RuleHelpXamlBuilder
+    /// <summary>
+    /// Information about a single XAML output element written to the new stream of XAML.
+    /// </summary>
+    internal struct XamlOutputElementInfo
     {
-        /// <summary>
-        /// Information about a single XAML output element written to the new stream of XAML.
-        /// </summary>
-        private struct XamlOutputElementInfo
+        public XamlOutputElementInfo(string htmlElementName, bool supportsInlines)
```

Comment:
removed it from inside the class so it can be reached from translator and generator